README.md

```diff
@@ -169,6 +170,17 @@ let onSubmit = (values, ~setError, ~setSubmitting) => {
 <Form schema onSubmit>
 ```
 
+### onFormStateChange param
+This hook if passed will be called in each case the form state changes, useful when you want to lift the form state for some reason to its parent.
```

Comment:
*some reasons

Reply:
but I think that for when the `some` will follow something that is unknown or does not have a fixed identity, it needs to be singular 🤔

Reply:
Hmm, I have no ideas :P

Reply:
haha me too, I just searched actually